Upstream-Status: Backport [22d12d74e7bc56b09cfb1973aa89ec8d714fa3fc]
From: Eastdong <31920925+IEAST@users.noreply.github.com>
Date: Thu, 23 Feb 2023 06:08:36 +0800
Subject: [PATCH] Add missing <atomic> content to fix gcc compilation for RISCV
 architecture. (#2211)

* Add missing <atomic> content to fix gcc compilation for RISCV architecture.

Co-authored-by: Moritz Firsching <firsching@google.com>
---
 lib/jxl/enc_xyb.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/jxl/enc_xyb.cc b/lib/jxl/enc_xyb.cc
index 1ef78877..2ee0abf8 100644
--- a/lib/jxl/enc_xyb.cc
+++ b/lib/jxl/enc_xyb.cc
@@ -6,6 +6,7 @@
 #include "lib/jxl/enc_xyb.h"
 
 #include <algorithm>
+#include <atomic>
 #include <cstdlib>
 
 #undef HWY_TARGET_INCLUDE
-- 
2.30.2

